ablySpec/Auth.swift

```diff
@@ -599,5 +599,164 @@ class Auth : QuickSpec {
                 expect(createTokenRequestMethodWasCalled).to(beTrue())
             }
         }
+
+        // RSA9
+        describe("createRequestToken") {
+
+            // RSA9a
+            it("should create and sign a TokenRequest") {
+                let options = AblyTests.commonAppSetup()
+                let rest = ARTRest(options: options)
+                let expectedClientId = "client_string"
+                let tokenParams = ARTAuthTokenParams(clientId: expectedClientId)
+
+                rest.auth.createTokenRequest(tokenParams, options: options, callback: { tokenRequest, error in
+                    expect(tokenRequest?.clientId).to(equal(expectedClientId))
+                    expect(tokenRequest?.mac).toNot(beNil())
+                    expect(tokenRequest?.nonce).toNot(beNil())
+                })
+            }
+
+            // RSA9b
+            it("should support AuthOptions") {
+                let rest = ARTRest(options: AblyTests.commonAppSetup())
+                let auth: ARTAuth = rest.auth
+
+                let authOptions = ARTAuthOptions(key: "key:secret")
+
+                auth.createTokenRequest(nil, options: authOptions, callback: { tokenRequest, error in
+                    expect(tokenRequest?.keyName).to(equal("key"))
+                })
```

Comment:
`queryTime` is the only property used on the `createTokenRequest` method and it is already addressed [here](https://github.com/ably/ably-ios/blob/267c9256ff38fc180b6cd7902d679d597435d5b9/ablySpec/Auth.swift#L745).